code/iaas/allocator/src/main/java/io/cattle/platform/allocator/constraint/ContainerLabelAffinityConstraint.java

```diff
@@ -11,7 +11,7 @@
 
 public class ContainerLabelAffinityConstraint implements Constraint {
     public static final String ENV_HEADER_AFFINITY_CONTAINER_LABEL = "affinity:container_label:";
-    public static final String LABEL_HEADER_AFFINITY_CONTAINER_LABEL = "io.rancher.scheduler." + ENV_HEADER_AFFINITY_CONTAINER_LABEL;
+    public static final String LABEL_HEADER_AFFINITY_CONTAINER_LABEL = "io.rancher.scheduler.affinity:container_label";
```

Comment:
@sonchang @vincent99 Just need a consensus.  Are we going with `container-label` or `container_label`. (same applies for host_label).  I void for `container-label`